champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/ranger/sword/WolfsPounce.java

```diff
@@ -132,7 +140,7 @@ public boolean shouldDisplayActionBar(Gamer gamer) {
 
     @Override
     public void trackPlayer(Player player, Gamer gamer) {
-        gamer.getActionBar().add(900, actionBarComponent);
+        gamer.getActionBar().add(50, actionBarComponent);
```

Comment:
Make it a higher priority. If we ever use the action bar for something more important it will mess with overwriting it.